kernel/packages/shared/renderer/sagas.ts

```diff
@@ -52,20 +51,29 @@ export function* ensureRenderer() {
 }
 
 function* initializeRenderer(action: InitializeRenderer) {
-  const { container, buildConfigPath } = action.payload
+  const { container } = action.payload
```

Comment:
```suggestion
  const { container, baseUrl } = action.payload
```

Please move the JSON from the HTML to here, create the final URL using `baseUrl` instead of `%PUBLIC_URL%`